### PR TITLE
MODSOURMAN-771 - New API information for DI Delete Logs

### PR DESCRIPTION
**Pull request by SatishkumarPurswani**

### CODE REVIEW

File: schemas/mod-source-record-manager/DeleteJobExecutionsDto.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Request to initialize JobExecution entities",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "ids": {
+      "description": "Information about files to upload",
```
Comment (RuslanLavrov) [inline schemas/mod-source-record-manager/DeleteJobExecutionsDto.json:8]: Hello @SatishkumarPurswani, could you please update the description according to DTO purpose. 
Something like this: "Id of JobExecution entities to be deleted".

Comment (RuslanLavrov) [inline schemas/mod-source-record-manager/DeleteJobExecutionsDto.json:8]: Hello @SatishkumarPurswani, could you please update the description according to DTO purpose. 
Something like this: "Id of JobExecution entities to be deleted".

File: schemas/mod-source-record-manager/DeleteJobExecutionsDto.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Request to initialize JobExecution entities",
```
Comment (RuslanLavrov) [inline schemas/mod-source-record-manager/DeleteJobExecutionsDto.json:3]:
```suggestion
  "description": "Request to delete JobExecution entities",
```
```suggestion
  "description": "Request to initialize JobExecution entities",
```

Comment (RuslanLavrov) [inline schemas/mod-source-record-manager/DeleteJobExecutionsDto.json:3]:
```suggestion
  "description": "Request to delete JobExecution entities",
```
```suggestion
  "description": "Request to initialize JobExecution entities",
```

**Review by AivarIUsupov — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

File: schemas/mod-source-record-manager/DeleteJobExecutionsDto.json (hunk 1 above)
Comment (SatishkumarPurswani) [reply schemas/mod-source-record-manager/DeleteJobExecutionsDto.json]: @RuslanLavrov - updated as per review comments. Thanks 🙂

**Review by SerhiiNosko — APPROVED** [review]

**Comment by okolawole-ebsco** [comment]
@SatishkumarPurswani please include me PR's such as this. Thanks!

**Pull request merged**